ChangeLog

```diff
@@ -27,6 +27,7 @@
 
 === Bug fixes ===
 
+ * #886 (problem with getSupport)
```

Comment:
could you squeeze this in the 1.9 part of the changelog so it merges nicely in the 1.9 branch ?